projectq/backends/_sim/_classical_simulator.py

```diff
@@ -0,0 +1,131 @@
+"""
```

Comment:
Missing license header:

Every new file should start with the same Apache license boiler plate.